search/management/commands/reindex_solr.py

```diff
@@ -18,20 +18,84 @@
 #     See AUTHORS file.
 #
 
+import logging
+import math
+
+from django.conf import settings
 from django.core.management.base import BaseCommand
+from utils.search.solr import Solr, SolrException
+from utils.text import remove_control_chars
+from utils.search.search_general import delete_sound_from_solr
 from sounds.models import Sound
-from utils.search.search_general import add_all_sounds_to_solr, delete_sound_from_solr
```

Comment:
did you delete this function from `search_general`? I don't think it's used anywhere else other than this command.